graylog2-web-interface/src/components/content-packs/ContentPackVersions.test.jsx

```diff
@@ -4,39 +4,39 @@ import { mount } from 'enzyme';
 import 'helpers/mocking/react-dom_mock';
 import URLUtils from 'util/URLUtils';
 
+import ContentPack from 'logic/content-packs/ContentPack';
+import ContentPackRevisions from 'logic/content-packs/ContentPackRevisions';
 import ContentPackVersions from 'components/content-packs/ContentPackVersions';
 
 describe('<ContentPackVersions />', () => {
   URLUtils.areCredentialsInURLSupported = jest.fn(() => { return false; });
-  const buildPack = (rev) => {
-    return {
-      id: '1',
-      rev: rev,
-      title: 'UFW Grok Patterns',
-      description: 'Grok Patterns to extract informations from UFW logfiles',
-      states: ['installed', 'edited'],
-      summary: 'This is a summary',
-      vendor: 'graylog.com',
-      url: 'www.graylog.com',
-    };
-  };
+  const contentPackRev = ContentPack.builder()
+    .id('1')
+    .name('UFW Grok Patterns')
+    .description('Grok Patterns to extract informations from UFW logfiles')
+    .summary('This is a summary')
+    .vendor('graylog.com')
+    .url('www.graylog.com');
+
   const contentPack = {
-    1: buildPack(1),
-    2: buildPack(2),
-    3: buildPack(3),
-    4: buildPack(4),
+    1: contentPackRev.rev(1).build().toJSON(),
```

Comment:
I think we don't need `toJSON` here either, right?

Reply:
You are absolutely right!!